doc/sphinx/credits.rst

```diff
@@ -1297,3 +1297,113 @@ system, is now upcoming and will rely on Inria’s newly created Foundation.
 | Paris, August 2017,
 | Matthieu Sozeau and the |Coq| development team
 |
+
+Credits: version 8.8
+====================
+
+
+|Coq| version 8.8 contains the result of refinements and stabilization of
+features and deprecations, cleanups of the internals of the system along
+with a few new features. The main user visible changes are:
+
+- Kernel: fix a subject reduction failure due to allowing fixpoints
+  on non-recursive values, which allows to recover full parametricity
+  for CIC, by Matthieu Sozeau. Handling of evars in the VM (the kernel
+  still does not accept evars) by Maxime Dénès.
+
+- Gallina: always use the projection printing style for primitive
+  projections by Maxime Dénès.
+
+- Notations: many improvements on recursive notations and
+  integration with pattern binding by Hugo Herbelin.
```

Comment:
The expression "integration with pattern binding" looks a bit enigmatic to me (but maybe it is only me?). However, it is not easy to find a concise alternative proposal. Maybe "support for patterns in the syntax of notations" (though I know that the word "pattern" has several meanings in the context of Coq and that it is not necessary clear about which patterns we are talking about).

Reply:
"support for destructuring patterns in the syntax of notations" ?

Reply:
Yup, good suggestion

Reply:
Good to me too.